cluster/gce/windows/testonly/install-ssh.psm1

```diff
@@ -0,0 +1,271 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+<#
+.SYNOPSIS
+  Library for installing and running Win64-OpenSSH. NOT FOR PRODUCTION USE.
+
+.NOTES
+  This module depends on common.psm1. This module depends on third-party code
+  which has not been security-reviewed, so it should only be used for test
+  clusters. DO NOT USE THIS MODULE FOR PRODUCTION.
+#>
+
+Import-Module -Force C:\common.psm1
+
+$OPENSSH_ROOT = 'C:\Program Files\OpenSSH'
+$USER_PROFILE_MODULE = 'C:\user-profile.psm1'
```

Comment:
Maybe we can put these modules in the same directory, but I'm okay since this gets invoked only for test clusters.

Reply:
You're right, sticking everything in C:\ might not be ideal. I'll consider this in future PRs.